internal/blueprint/f30_helpers.go

```diff
@@ -33,7 +33,7 @@ func getF30BuildPipeline() *pipeline.Pipeline {
 
 func getF30Pipeline() *pipeline.Pipeline {
 	p := &pipeline.Pipeline{
-		BuildPipeline: getF30BuildPipeline(),
+		BuildPipeline: nil, //getF30BuildPipeline(), because we can't use buildroot withou libdir, see osbuild repo for more info
```

Comment:
This is a bug we must fix in osbuild, we cannot have images without build pipelines, so I would not comment this out.

Reply:
In that case: https://github.com/osbuild/osbuild/pull/145

Reply:
Will look into that. Thank!